.github/workflows/release.yaml

```diff
@@ -0,0 +1,25 @@
+name: Release
+on:
+  push:
+    branches:
+      - master
+jobs:
+  release:
+    name: Release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: '8.x'
+      - name: Install dependencies
+        run: yarn install
+      - name: Release
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+        run: yarn run semantic-release
```

Comment:
yep, basically, they had only one config: it ran in PR merge and master merges.
The plugin they used `condition-circle` basically checked if it was a master merge and allowed the release to happen, in all other cases the plugin did nothing at all.

I replaced it with a `master` only release workflow, and since I noticed it did not even do "validation" of anything I removed it from the build run since it really just does not add anything.

Reply:
ok! perfect then